bot.py

```diff
@@ -12,6 +14,30 @@
 # target_channel_id = 722609125950750771 # testing
 activity = True
 
+@bot.event
+async def on_message(message):
+    if message.author == client.user:
+        return
+    match = re.search((?:\b|\s+)(?:ope)(?:\s+|\b), message.content.lower)
+    if match:
+        with open('ope.csv', 'w', newline='') as csvfile:
+            reader = csv.reader(csvfile, delimiter=',', quotechar='|')
+            rows = []
+            author_found = False
+            author_count = 0
+            for row in reader:
+                if (message.author == row[0]):
+                    row[1] += 1
+                    author_found = True
+                    author_count = row[1]
+                rows.append[row]
+            if (author_found == False):
```

Comment:
```suggestion
            if not author_found:
```
I feel this is easier to read :)

Reply:
def more pythonic